app/src/main/java/net/squanchy/eventdetails/widget/EventDetailsLayout.java

```diff
@@ -51,7 +51,7 @@ public void updateWith(Event event) {
         placeView.setVisibility(event.placeVisibility());
         placeView.setText(event.place());
         speakersView.setVisibility(event.speakersVisibility());
-        speakersView.setText(event.speakers());
+        speakersView.setText(event.speakersNames());
```

Comment:
Just as a reminder, we have to add them in the database, from the JSON, we have the indexes of the speakers